docs/CODE_OF_CONDUCT.rst

```diff
@@ -1,127 +1,148 @@
+  overall community
+
+Examples of unacceptable behavior include:
+
+- The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
+- Trolling, insulting or derogatory comments, and personal or political
+  attacks
+- Public or private harassment
+- Publishing others’ private information, such as a physical or email
+  address, without their explicit permission
+- Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+Enforcement Responsibilities
+----------------------------
+
+Community leaders are responsible for clarifying and enforcing our
+standards of acceptable behavior and will take appropriate and fair
+corrective action in response to any behavior that they deem
+inappropriate, threatening, offensive, or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other
+contributions that are not aligned to this Code of Conduct, and will
+communicate reasons for moderation decisions when appropriate.
 
 Scope
 -----
 
-This Code of Conduct applies both within project spaces and in public
-spaces when an individual is representing the project or its community.
-Examples of representing a project or community include using an
-official project e-mail address, posting via an official social media
-account, or acting as an appointed representative at an online or
-offline event. Representation of a project may be further defined and
-clarified by project maintainers.
+This Code of Conduct applies within all community spaces, and also
+applies when an individual is officially representing the community in
+public spaces. Examples of representing our community include using an
+official e-mail address, posting via an official social media account,
+or acting as an appointed representative at an online or offline event.
 
 Enforcement
 -----------
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may
-be reported by contacting Nick Murphy at namurphy@cfa.harvard.edu or any
-member of the Coordinating Committee. All complaints will be reviewed
-and investigated and will result in a response that is deemed necessary
-and appropriate to the circumstances. The project team is obligated to
-maintain confidentiality with regard to the reporter of an incident.
-Project team members should recuse themselves from enforcement of the
-code of conduct for a given incident if they have an actual or apparent
-conflict of interest.  Further details of specific enforcement policies
-may be posted separately.
-
-Project maintainers who do not follow or enforce the Code of Conduct in
-good faith may face temporary or permanent repercussions as determined
-by other members of the project’s leadership.
+be reported to the community leaders responsible for enforcement at
+conduct@plasmapy.org. All complaints will be reviewed and
```

Comment:
It turns out that Sphinx will render email addresses correctly with a `mailto:` prefix if they show up in plain text.